mlflow/protos/service.proto

```diff
@@ -918,6 +936,20 @@ message GetRun {
   }
 }
 
+message ListAllColumns {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+
+  // List of experiment IDs to search over.
+  required string experiment_id = 1;
```

Comment:
This API is very specific/custom API for showing columns in UI and not particularly generalizable over multiple experiments ... etc

Reply:
Mlflow today doesn't support comparison between different experiments.
This PR follows this pattern, is this something that's meant to be changed in mlflow in the future ?